packages/@aws-cdk/aws-s3-deployment/lib/bucket-deployment.ts

```diff
@@ -165,6 +166,14 @@ export interface BucketDeploymentProps {
    */
   readonly serverSideEncryptionCustomerAlgorithm?: string;
 
+  /**
+   * Sets the ACL for the object when the command is performed.
```

Comment:
Should this be "objects in this bucket"?

Reply:
I have updated docblock to match other properties related to system metadata.